txdbus/test/test_authentication.py

```diff
@@ -54,44 +61,44 @@ def test_bad_auth_message(self):
         self.assertRaises(DBusAuthenticationFailed, self.send, 'BAD_LINE')
 
     def test_rejection(self):
-        self.ae(self.ca.authMech, 'EXTERNAL')
-        self.are( 'AUTH EXTERNAL')
+        self.ae(self.ca.authMech, b'EXTERNAL')
+        self.are(b'AUTH EXTERNAL')
         self.send('REJECTED')
```

Comment:
Mostly because I'm not done yet (`test_authentication`, in particular, in only halfway through).